server/src/main/java/com/google/coffeehouse/common/MembershipConstants.java

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.coffeehouse.common;
+
+/**
+* The MembershipConstants class holds constant values that denote membership status.
+*/
```

Comment:
Just wanted to echo that having a separate class for constants is a great architecture idea! Nice!

Reply:
Thanks - It was actually Sarah's suggestion, so thank you to Sarah!